app/src/main/res/drawable/ic_audio_streaming_on_24dp.xml

```diff
@@ -1,6 +1,7 @@
 <vector
   xmlns:android="http://schemas.android.com/apk/res/android"
   android:width="24dp" android:height="24dp"
+  android:autoMirrored="true"
```

Comment:
This should not be reversed. Only icons which represent direction should be reversed.

Reply:
Are you sure that we need not reverse this?

Reply:
I have checked it by following these steps:
1. Try to find any app where we can see this icon getting used.
2. Take screenshot in LTR
3. Using `Developer Options`, change to RTL forcefully.
4. Now take screenshot of same previous screen in RTL.
5. Compare these screenshots for the icon to decide if mirroring is required or not.


Because of this I have mentioned that this should not be reversed.

Reply:
Okay got it. Thanks for this info.